<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QWidget" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter title to search for:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SearchlineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="SearchUpdate">
   <property name="geometry">
    <rect>
     <x>227</x>
     <y>20</y>
     <width>171</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Search for Article</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ViewArticlesButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>View articles</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ViewArticleslineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="viewarticlesupdate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="latestarticlesButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>View latest articles</string>
   </property>
  </widget>
  <widget class="QLabel" name="latestarticlesupdate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>231</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="ArticleRatinglineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ArticleRatingnumberlineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="RateButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="RateUpdate">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>171</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save To file</string>
   </property>
  </widget>
  <widget class="QLabel" name="SaveUpdate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>420</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
